evalml/objectives/objective_base.py

```diff
@@ -73,28 +76,39 @@ def _standardize_input_type(y_in):
         """
         if isinstance(y_in, (pd.Series, pd.DataFrame)):
             return y_in.to_numpy()
+        if isinstance(y_in, ww.DataColumn):
+            return _convert_woodwork_types_wrapper(y_in.to_series()).to_numpy()
         return y_in
 
     def validate_inputs(self, y_true, y_predicted):
         """Validates the input based on a few simple checks.
 
         Arguments:
-            y_predicted (pd.Series): Predicted values of length [n_samples]
+            y_predicted (pd.Series, pd.DataFrame, ww.DataColumn, or ww.DataTable): Predicted values of length [n_samples]
             y_true (pd.Series): Actual class labels of length [n_samples]
 
         Returns:
             None
         """
-        if len(y_predicted) != len(y_true):
+        is_multidimensional = False
```

Comment:
If score needs predict_proba, we can have a multidimensional array to check, so updating. Not sure why this wasn't caught previously 🤔

Reply:
Ooo, interesting. I see, since in a lot of cases we use X_y_binary which results in numpy input, maybe our tests previously missed this?